.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```

Comment:
We shouldn't check these in. Also, recommend using Rust Analyzer in VSCode instead!